foundations/foundation-common/src/test/java/org/apache/servicecomb/foundation/common/utils/TestJsonUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.servicecomb.foundation.common.utils;
+
+import org.junit.Test;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.Period;
+
+public class TestJsonUtils {
+    @Test
+    public void testJSR310() throws Exception {
+        String period = "\"P1Y1M2D\"";
+        String duration = "\"P1DT1H2M3S\"";
+        String offsetDateTime = "\"2018-05-28T11:00:00+08:00\"";
+        String localDate = "\"2018-05-28\"";
+        String localDateTime = "\"2018-05-28T11:00:00\"";
+
+        JsonUtils.OBJ_MAPPER.readValue(period, Period.class);
```

Comment:
I think we need to verify the value of object here.

Reply:
Ok, it makes sense.